frontend/src/lib/components/TaxonomicFilter/infiniteListLogic.ts

```diff
@@ -119,11 +122,11 @@ export const infiniteListLogic = kea<infiniteListLogicType>([
 
                     if (!remoteEndpoint) {
                         // should not have been here in the first place!
-                        return createEmptyListStorage(searchQuery)
+                        return createEmptyListStorage(swappedInQuery || searchQuery)
```

Comment:
the fix in the revert here is that now swappedInQuery can be undefined rather than always falling back to the provided searchQuery